src/stacks-hierarchy/Root_TicketAssistantStack/Root_TicketAssistantStack.tsx

```diff
@@ -0,0 +1,92 @@
+import React, {useState} from 'react';
+import {SafeAreaView} from 'react-native-safe-area-context';
+import {
+  createMaterialTopTabNavigator,
+  MaterialTopTabBarProps,
+} from '@react-navigation/material-top-tabs';
+import {PageIndicator} from '@atb/components/page-indicator';
+import {StyleSheet} from '@atb/theme';
+
+import {StaticColorByType} from '@atb/theme/colors';
+import {TicketAssistant_WelcomeScreen} from '@atb/stacks-hierarchy/Root_TicketAssistantStack/TicketAssistant_WelcomeScreen';
+import {TicketAssistantStackParams} from '@atb/stacks-hierarchy/Root_TicketAssistantStack/navigation-types';
+import {TicketAssistant_FrequencyScreen} from '@atb/stacks-hierarchy/Root_TicketAssistantStack/TicketAssistant_FrequencyScreen';
+import {TicketAssistant_CategoryPickerScreen} from '@atb/stacks-hierarchy/Root_TicketAssistantStack/TicketAssistant_CategoryPickerScreen';
+import {TicketAssistant_DurationScreen} from '@atb/stacks-hierarchy/Root_TicketAssistantStack/TicketAssistant_DurationScreen';
+import {TicketAssistant_ZonePickerScreen} from '@atb/stacks-hierarchy/Root_TicketAssistantStack/TicketAssistant_ZonePickerScreen';
+import {TicketAssistant_SummaryScreen} from '@atb/stacks-hierarchy/Root_TicketAssistantStack/TicketAssistant_SummaryScreen';
+import {FullScreenHeader} from '@atb/components/screen-header';
+import {RootStackScreenProps} from '@atb/stacks-hierarchy';
+
+const Tab = createMaterialTopTabNavigator<TicketAssistantStackParams>();
+const themeColor: StaticColorByType<'background'> = 'background_accent_0';
+
+type Props = RootStackScreenProps<'Root_TicketAssistantStack'>;
+
+export const Root_TicketAssistantStack = ({navigation}: Props) => {
+  const styles = useStyles();
+  const [activeTab, setActiveTab] = useState(0);
+  const [lastTab, setLastTab] = useState<any>();
+  return (
+    <>
+      {activeTab !== 0 ? (
+        <FullScreenHeader
+          leftButton={{
+            type: 'back',
+            //Navigate to previous tab
+            onPress: () => {
+              navigation.navigate(lastTab);
```

Comment:
```suggestion
              navigation.navigate(previousTab);
```
lastTab or previousTab? I see that you wrote previous tab in comment above

Reply:
Renamed it👌🏻